react/hooks/useSetFlagshipUi/useSetFlagshipUI.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react'
+import pickBy from 'lodash/pickBy'
+import identity from 'lodash/identity'
+import isEmpty from 'lodash/isEmpty'
+import isObject from 'lodash/isObject'
+
+import { useWebviewIntent } from 'cozy-intent'
+import { FlagshipUI } from 'cozy-intent/dist/api/models/applications'
+import { WebviewService } from 'cozy-intent/dist/api/services/WebviewService'
+
+const parseArg = (
+  webviewIntent?: WebviewService | void,
+  arg?: FlagshipUI
```

Comment:
is it possible to call parseArg(undefined,arg) ?

if so, why not using ({ webviewIntent, arg }) ?

Reply:
nothing to do then